molecule/default/converge.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  pre_tasks:
+    - name: Update apt cache.
+      apt: update_cache=true cache_valid_time=600
+      when: ansible_os_family == 'Debian'
+
+  roles:
+    - role: ansible-role-ssh-user
```

Comment:
Can we add some vars here so that we test some role functionality?

Reply:
Done.
```suggestion
    - role: ansible-role-ssh-user
```
```suggestion
    - role: ansible-role-ssh-user
```
![ssh-molecule](https://user-images.githubusercontent.com/63741367/99390344-7aae9c80-28fe-11eb-9dd6-14e4b9414dbc.png)